src/components/search/results/partial-results.vue

```diff
@@ -15,17 +15,17 @@
 </template>
 
 <script lang="ts">
-  import { PartialResultsList, PartialQueryButton } from '@empathyco/x-components/search';
   import { BaseGrid } from '@empathyco/x-components';
   import { Component, Vue } from 'vue-property-decorator';
-  import ResultComponent from './result.vue';
+  import { PartialQueryButton, PartialResultsList } from '@empathyco/x-components/search';
+  import ResultComponent from '../../results/result.vue';
```

Comment:
I guess we can rename this to `Result` and avoid renaming it later when registering it in `line 26`? I know it is not a big deal and it was like that before, just saying 😅

Reply:
I'm not sure about this to be honest, I still find confusing the component names on the archetype and distinguish when it is coming from x or is just a component for the archetype. Anyway, in this case I think ESLint will complain since there is already a Result on RestultComponent file.